src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,23 @@
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
 // Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    <ul>
+      {todos.map(todo => (
+        <li className="TodoList__item">
+          <TodoInfo
+            key={todo.id}
```

Comment:
Put id to the parent container
```suggestion
        <li className="TodoList__item" key={todo.id}>
          <TodoInfo
```